travis/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+
+if [[ "$TRAVIS_TAG" =~ ^release-[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+?$ ]]; then
+    echo "We are on release- tag"
+    echo "bumping versions and creating vx.x.x tag"
+    echo "final artifact will be published in build for the tag"
+    version=$(echo "${TRAVIS_TAG}" | sed 's/^release-//')
+    ./gradlew release release -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=${version}
```

Comment:
is  this what makes the commits? Do we need to set up git identity for the bot?

Reply:
travis creates commits, we already inject GH token there.

Reply:
Yes the gradle plugin creates the commit, I will verify in my fork if it works with our current config. If no I will add required credentials.